app/models/conference.rb

```diff
@@ -12,12 +12,16 @@ class Conference < ActiveRecord::Base
   scope :ordered, ->(sort = {}) { order([sort[:order] || 'starts_on, name', sort[:direction] || 'asc'].join(' ')) }
   scope :in_current_season, -> { where(season: Season.current) }
 
+  def dates
```

Comment:
So, looking at the latter usage, this method only serves as a view helper and is never used solo, but *always* concatinated with `#compact`. So why offering direct access to `dates` at all, if it's never actually used? This can eventually make things more complex later on... imagine someone forgets to call `compact` - nothing useful will be rendered.

I would suggest to just rename this method to `date_range` and directly return the compact version of it.

This can of course be done in multiple ways:
a) call the public method in the model
```ruby
DateRange.new(start_date: starts_on, end_date: ends_on).compact
```

b) directly return the compact version, e.g.
```ruby
class DateRange
  # ...
  def self.call(start_date:, end_date:)
    self.new(start_date: start_date, end_date: end_date).compact
  end
end

# in the model:
DateRange.call(start_date: starts_on, end_date: ends_on)
```